spec/predicates/simple-verification.md

```diff
@@ -0,0 +1,109 @@
+# Predicate type: Simple Verification
+
+Type URI: https://in-toto.io/attestation/simple-verification/v0.1
+
+Version: 0.1
+
+Authors: Tom Hennen (@TomHennen), Andrew McNamara (@arewm)
+
+## Purpose
+
+A Simple Verification Attestation (SVA) communicates that an artifact has been evaluated against one or more policies, and records the properties that were verified, at the point-in-time of the evaluation. This enables consumers to trust a concise summary of verification results, without requiring access to all underlying attestations or policies.
+
+## Use Cases
+
+-   Allowing software consumers to delegate complex policy decisions to a trusted verifier, and rely on a simple summary of which properties were verified.
+-   Supporting evolving verification needs (e.g., SLSA, vulnerability scanning, organizational policies) without coupling the SVA format to any specific framework.
+-   Enabling verifiers to add additional information (e.g., policy references, expiration, property-level timing) as needed, while maintaining a simple core schema.
+
+## Prerequisites
+
+This predicate depends on the [in-toto Attestation Framework]. Familiarity with the concept of attestation statements and the use of URIs for identifying verifiers and resources is assumed.
+
+## Model
+
+An SVA is an attestation that a verifier (`verifier.id`) has checked an artifact (`resourceUri`) at a specific time (`timeCreated`) and asserts a set of verified properties (`properties`). The SVA does not include information for reproducing the verification result. Instead, verifiers MAY generate additional provenance attestations for the simple verification attestation as needed.
+
+## Schema
+
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/simple-verification/v0.1",
+  "predicate": {
+    "verifier": {
+      "id": "<URI>"
+    },
+    "timeCreated": "<TIMESTAMP>",
+    "resourceUri": "<artifact-URI>",
```

Comment:
Perhaps I'm missing something, but isn't `resourceUri` essentially the attestation subject?  Also, should this field be a resource descriptor?

> An SVA is an attestation that a verifier (`verifier.id`) has checked an artifact (`resourceUri`)

Reply:
> Perhaps I'm missing something, but isn't `resourceUri` essentially the attestation subject? Also, should this field be a resource descriptor?
> 
> > An SVA is an attestation that a verifier (`verifier.id`) has checked an artifact (`resourceUri`)

IMO they are orthogonal.

The subject records info relevant for the artifact being verified; the "resourceUri", OTOH, identifies the protection domain the artifact is verified against.

For example,
- The subject may have a uri "sftp://akamai.net/foo/bar.zip" -- that is where it is retrieved from;
- The SVA resourceUri may be "https://foo.com/app_bar" -- that is the service the artifact is going to be deployed onto (hence it needs to be verified against a policy for this URI).

Reply:
I'm going to resolve this thread as the field is gone. Please provide separate specific feedback if you think that we should add it back.